src/go/rpk/pkg/api/admin/api_partition.go

```diff
@@ -44,3 +46,9 @@ func (a *AdminAPI) GetPartition(
 		nil,
 		&pa)
 }
+
+// Reconfigurations returns the list of ongoing partition reconfigurations.
```

Comment:
Prior commit documented the type, this one the method. I think we pretty consistently document the type, so we should here

perhaps the prior commit should _also_ document the methods, though.

Reply:
Fixed :+1: